docker/.env

```diff
@@ -0,0 +1,11 @@
+HOST_PUDL_IN=./pudl_in
+HOST_PUDL_OUT=./pudl_out
+CONTAINER_HOME=/home/catalyst
+CONTAINER_PUDL_IN=/home/catalyst/pudl_in
+CONTAINER_PUDL_OUT=/home/catalyst/pudl_out
+CONDA_PREFIX=/home/catalyst/env
+PUDL_SETTINGS_YML=src/pudl/package_data/settings/etl_full.yml
```

Comment:
The mix of absolute and relative paths seems like it could get us in trouble.

Reply:
I made `docker/.env` an absolute path.